Makefile

```diff
@@ -24,4 +24,5 @@ develop: clean
 check:
 	. venv/bin/activate && \
 	flake8 dashdotdb && \
+	mypy --ignore-missing-imports --install-types --non-interactive --no-implicit-optional && \
```

Comment:
I prefer explicitly ignoring the missing imports like we're doing here: https://github.com/app-sre/qontract-reconcile/blob/master/setup.cfg#L11

It makes us more aware when a new dependency is added and it doesn't have types, as well as providing a list of packages to periodically review to see if types have been added in newer versions. With that being said, this isn't a blocking change. If you do want to make this change, you can just do `pip freeze | grep 'types\|stub'` and add those dependencies to requirements-check.txt, and then just an entry per package in setup.cfg that is still throwing missing import errors.